File: prime.cpp

#include <stdio.h>
#include <iostream>
#include <ostream>
#include <chrono>


int main(){
    int targetNumber=1000;
    auto start = std::chrono::high_resolution_clock::now();
    for (int i=0;i<targetNumber;i++){
        int factors=0;
        for (int x=1;x<i+1;x++){
            if (i%x==0){
                factors++;
            }
        }
        if (factors==2){
            std::cout<<i<<std::endl;
        }
    }
    auto stop = std::chrono::high_resolution_clock::now();
    auto duration = std::chrono::duration_cast<std::chrono::microseconds>(stop - start);

    std::cout<<duration.count()<<" microseconds"<<std::endl;
    return 0;
}

File: prime.py

import time
targetNumber=1000
start_time = time.time()

for i in range(0,targetNumber+1):
    factors=0
    for x in range(1, i+1):
        if i%x==0:
            factors+=1
    if factors==2:
        print(i)



print("%s microseconds" % ((time.time() - start_time)*(1000000)))
